packages/vue/src/components/FInteractiveTable/FInteractiveTable.vue

```diff
@@ -595,6 +593,16 @@ function setActiveRow(row: T | undefined): void {
                             * `row: ListItem;` The object to be visualized.
                         -->
                         <slot v-bind="{ row }" />
+
+                        <!--
+                            @slot Slot for screen reader text when row get focus. The row object is available through `v-slot="{ <propertyName> }"`, e.g. `v-slot="{ row }"`.
+
+                            The following properties are available:
+
+                            * `row: ListItem;` The object to be visualized.
+                            @binding {T} row - The object to be screen read.
+                        -->
+                        <slot name="row-description" v-bind="{ row }" class="sr-only"></slot>
```

Comment:
Fungerar det att sätta `class` direkt på `<slot>`? (Jag vet inte)

Reply:
Men den hamnar fortfarande direkt under `<tbody>` gör den inte? Det är inte tillåtet per [standarden](https://html.spec.whatwg.org/multipage/tables.html#the-tbody-element).